src/ocm/reducers/clusters/clustersSlice.ts

```diff
@@ -21,6 +21,7 @@ export const fetchClustersAsync = createAsyncThunk<Cluster[] | void>(
       handleApiError(e, () => {
         void Promise.reject('Failed to fetch clusters.');
       });
+      return rejectWithValue(value);
```

Comment:
Would it work removing this line and updating?
```

 return handleApiError(e, () => {
        return Promise.reject(value);
      });
});
```
Or alternatively, with the code:
```
handleApiError(e);
return rejectWithValue(value);
```

either way, I think it's best not to duplicate the logic for rejecting the promise, while making sure it will be rejected, or the `POLLING_ERROR` will not be registered.

Reply:
@celdrake WDYT? 
In ocm/reducers/clusters/currentClusterSlice.ts, the code is:

```
return handleApiError(e, () => {
      return rejectWithValue({
        code: getApiErrorCode(e as Error | AxiosError),
        message: getApiErrorMessage(e),
      });
    });

Reply:
It's fine with me if it works as it should. The main idea is to reject the promise only once.
I can't say what the effect is of rejecting the the `code/message` object instead of `value`. I thought `value` was the message we wanted to display.

Reply:
@jgyselov can you try it?

Reply:
I've settled on 
```
return handleApiError(e, () => rejectWithValue(value));
```
(we also don't really care what `value` is).

Reply:
Great, thanks!